include/veriblock/storage/block_repository_rocks.hpp

```diff
@@ -171,31 +195,23 @@ class BlockRepositoryRocks : public BlockRepository<Block> {
     return true;
   }
 
-  // decode hashes blob
-  std::set<hash_t> hashesFromString(const std::string& hashesData) const {
-    if (hashesData.size() % sizeof(hash_t)) {
-      throw db::DbError("Not parceable hashes blob");
-    }
-
-    std::set<hash_t> result{};
-    for (size_t i = 0; i < (hashesData.size() / sizeof(hash_t)); i++) {
-      std::string blobPartStr =
-          hashesData.substr(i * sizeof(hash_t), (i + 1) * sizeof(hash_t));
+  void clear() override { 
+    // call BlockRepositoryRocksManager.clear() instead
+    return;
+  }
 
-      result.insert(Blob<32>(blobPartStr));
-    }
-    return result;
+  std::unique_ptr<WriteBatch<stored_block_t>> newBatch() override {
+    return std::make_unique<WriteBatchRocks<stored_block_t>>(_db,
```

Comment:
make unique is C++14 feature https://en.cppreference.com/w/cpp/memory/unique_ptr/make_unique

Use `std::unique_ptr<T>(new T)`

Reply:
+